scripts/cut_release_or_hotfix_branch.py

```diff
@@ -146,6 +164,38 @@ def _verify_target_version_is_consistent_with_latest_released_version():
         assert int(curr_patch) == 0
 
 
+def _verify_hotfix_number_is_one_ahead_of_previous_hotfix_number(
+        remote_alias):
+    """Checks that the hotfix number is one ahead of previous hotfix
+        number.
+
+    Args:
+        remote_alias: str. The alias that points to the remote oppia
+            repository. Example: When calling git remote -v, you get:
+            upstream    https://github.com/oppia/oppia.git (fetch),
+            where 'upstream' is the alias that points to the remote oppia
+            repository.
+
+    Raises:
+        Exception: The difference between two continuous hotfix numbers
+             is not one.
+    """
+    all_branches = subprocess.check_output([
+        'git', 'branch', '-a'])[:-1].split('\n')
+
+    last_hotfix_number = 0
+    hotfix_branch_name_regex = '^%s/release-%s-hotfix-\\d*$' % (
+        remote_alias, TARGET_VERSION)
+    for branch_name in all_branches:
+        branch_name = branch_name.lstrip().rstrip()
+        if re.match(hotfix_branch_name_regex, branch_name):
+            hotfix_number = int(branch_name[branch_name.rfind('-') + 1:])
+            if hotfix_number > last_hotfix_number:
+                last_hotfix_number = hotfix_number
+
+    assert HOTFIX_NUMBER == last_hotfix_number + 1
```

Comment:
Can't we set the number automatically, instead of checking if someone put in correct number?

Reply:
Yes, we can do that based on fetching the all branches from remote and creating a new branch by incrementing the last number by 1. But I feel, the issue is there is that user might do a mistake of creating a new branch even if some old branch is present and changes can be pushed through that. So, the script checks this before creating a branch based on the number which user wants to create.

Reply:
I think for safety let's do manual input.